# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ShareableCanvasRenderer.cpp
# Commit: b15d91e64a25
# Full Hash: b15d91e64a2589d47ad023d61dbd45135669eb60
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - NON_PREMULT TextureClient iff NON_PREMULT CanvasClient. r=handyman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78800
# ==============================================================================

diff -r 71ad2ed8e9ba -r b15d91e64a25 gfx/layers/ShareableCanvasRenderer.cpp
--- a/gfx/layers/ShareableCanvasRenderer.cpp	Wed Jun 10 22:21:07 2020 +0000
+++ b/gfx/layers/ShareableCanvasRenderer.cpp	Wed Jun 10 22:21:09 2020 +0000
@@ -159,7 +159,8 @@
 
       const RefPtr<DrawTarget> dt = tc->BorrowDrawTarget();
 
-      const auto borrowed = BorrowSnapshot();
+      const bool requireAlphaPremult = false;
+      const auto borrowed = BorrowSnapshot(requireAlphaPremult);
       if (!borrowed) return nullptr;
 
       dt->CopySurface(borrowed->mSurf, {{0, 0}, size}, {0, 0});